contrib/check-config.sh

```diff
@@ -157,6 +157,14 @@ echo 'Generally Necessary:'
 echo -n '- '
 if [ "$(stat -f -c %t /sys/fs/cgroup 2> /dev/null)" = '63677270' ]; then
 	echo "$(wrap_good 'cgroup hierarchy' 'cgroupv2')"
+	cgroupv2_controller_loc="/sys/fs/cgroup/cgroup.controllers"
+	# Prints available cgroupv2 controllers if the file exists
+	if test -f "$cgroupv2_controller_loc"; then
+		mapfile -t cgroupv2_controllers < $cgroupv2_controller_loc
+		wrap_good "- cgroupv2 controllers" "${cgroupv2_controllers[*]}"
+	else
+		echo "$cgroupv2_controller_loc not found."
+	fi
```

Comment:
This file (fairly consistently) uses `camelCase` for variables (and `snake_case` for functions).  I think we should also consider it a failure case if we don't find the referenced file:

```suggestion
	cgroupv2ControllersFile='/sys/fs/cgroup/cgroup.controllers'
	if [ -s "$cgroupv2ControllersFile" ]; then
		mapfile -t cgroupv2Controllers < "$cgroupv2ControllersFile"
		wrap_good '- cgroupv2 controllers' "${cgroupv2Controllers[*]}"
	else
		wrap_warning "warning: missing '$cgroupv2ControllersFile'"
	fi
```

Reply:
My bad, I'll update the variable names to camelCase and create the warning if the file doesn't exist.